internal/controllers/clusterextension_controller.go

```diff
@@ -205,9 +205,6 @@ func (r *ClusterExtensionReconciler) reconcile(ctx context.Context, ext *ocv1alp
 	l.Info("getting installed bundle")
 	installedBundle, err := r.InstalledBundleGetter.GetInstalledBundle(ctx, ext)
 	if err != nil {
-		setInstallStatus(ext, nil)
-		// TODO: use Installed=Unknown
-		setInstalledStatusConditionFailed(ext, err.Error())
```

Comment:
I'm not sure I agree with this set of removals.

If we fail to get the installed bundle, that seems like a pretty direct signal for `Installed=Unknown`. In this case, something is majorly wrong with our ability to contact the api server, or we got an ephemeral error, and we'll hopefully set this back to `Installed=True` when the ephemeral issue goes away. For this, I can see an argument for holding off on setting `Installed=Unknown` until we see the same error N times in a row so that we avoid flapping, but it seems like we should eventually set `Installed=Unknown` if we truly can't get the installed bundle.

If we get `installedBundle==nil`, that's a clear signal for `Installed=False`. In this case, it seems like we could set the Installed condition to False if we get any other subsequent errors during reconcile.